docs/DeveloperGuide.adoc

```diff
@@ -844,16 +945,461 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Auto-fill
+
+*MSS*
```

Comment:
Remember to update all references to `AddressBook` to your application's name!

Reply:
Noted